docs/csharp/language-reference/compiler-messages/toc.yml

```diff
@@ -1701,10 +1695,6 @@ items:
     href: cs8373.md
   - name: CS8374
     href: cs8374.md
-  - name: CS8400
-    href: cs8400.md
-  - name: CS8401
-    href: cs8401.md
```

Comment:
Should these remain and let the redirect take over? I guess it's possible to be in the TOC looking for my error number.

Reply:
I've been removing them, and relying on the display Name filter in the TOC.